xdsl/dialects/linalg.py

```diff
@@ -601,6 +601,45 @@ def parse(cls, parser: Parser) -> Self:
         return transpose
 
 
+@irdl_op_definition
+class MatMulOp(IRDLOperation):
```

Comment:
Even though it stands for Matrix Multiplication, it sort of has become its own thing, and is capitalized `MatmulOp` in MLIR